include/grpc++/security/server_credentials.h

```diff
@@ -39,6 +39,7 @@
 
 #include <grpc++/security/auth_metadata_processor.h>
 #include <grpc++/support/config.h>
+#include <grpc/grpc_security.h>
```

Comment:
OK, I think that it is actually a good idea. Let's have a new header called grpc_security_constants.h where we put this new enum along with all the #define'd string constants for the auth context and the enum for the ssl root override.